Automattic/wp-calypso
# A4A > Marketplace: Add VIP content for the new hosting page

Closes https://github.com/Automattic/jetpack-genesis/issues/453
Closes https://github.com/Automattic/jetpack-genesis/issues/454

## Proposed Changes

This PR:

- Adds VIP hosting capabilities 
- Adds VIP hosting testimonials 


## Why are these changes being made?

* For the new hosting page

## Testing Instructions

* Open the A4A live link
* Go to Marketplace > Hosting > Click the Enterprise tab > Verify the UI looks as shown below:

![screencapture-agencies-localhost-3000-marketplace-hosting-2024-07-26-11_59_00](https://github.com/user-attachments/assets/5de52426-df3e-451e-ad86-3c5966978c0b)



## Pre-merge Checklist
